src/core_plugins/kibana/public/visualize/listing/visualize_listing_table.js

```diff
@@ -142,10 +142,21 @@ export class VisualizeListingTable extends Component {
   }
 
   renderRowCells(item) {
+
+    let flaskHolder;
+    if (item.type.isExperimental) {
+      flaskHolder = <span className="kuiIcon fa-flask ng-scope">&nbsp;</span>;
+    }else{
+      flaskHolder = <span />;
+    }
+
     return [
-      <a className="kuiLink" href={this.getUrlForItem(item)}>
-        {item.title}
-      </a>,
+      <span>
+        {flaskHolder}
+        <a className="kuiLink" href={this.getUrlForItem(item)}>
+          {item.title} foobariontos
```

Comment:
!["foobariontos" all our visualizations meme](https://user-images.githubusercontent.com/877229/32823624-23f341d4-c9de-11e7-943c-6998f81ebadd.jpg)

Reply:
ouch.... the sham